packages/editor/src/components/collab-sidebar/comment-form.js

```diff
@@ -31,13 +32,12 @@ function CommentForm( { onSubmit, onCancel, thread, submitButtonText } ) {
 
 	return (
 		<>
-			<TextareaControl
-				__next40pxDefaultSize
-				__nextHasNoMarginBottom
+			<TextareaAutosize
 				value={ inputComment ?? '' }
-				onChange={ setInputComment }
+				onChange={ ( comment ) =>
+					setInputComment( comment.target.value )
+				}
 				label={ __( 'Comment' ) }
```

Comment:
A couple of notes:

* The `TextareaAutosize` doesn't handle rendering labels for us. Let's handle it ourselves - `<VisuallyHidden as="label" htmlFor={ inputId }>`.
* I think we can use `rows={ 4 }` to alter minimum visible. There's also `maxRows`, which allows us to set an upper limit.

Reply:
- added rows= { 4 }
- set maxRows to 20
- added label using VisuallyHidden component (does that look right?)

Reply:
That looks right, thank you, @adamsilverstein!

We should also remove the leftover `label` prop from `TextareaAutosize` since it does nothing.

Reply:
> We should also remove the leftover label prop from TextareaAutosize since it does nothing 

right!